docs/src/Components.jsx

```diff
@@ -210,6 +210,21 @@ var Components = React.createClass({
           </ul>
         </article>
 
+        <article id="weather">
+          <h3>Weather</h3>
+          <p>Display weather information in a standard format. Combine a few of these components to create weather patterns i.e a 5 day forecast</p>
+          <CustomComponent codeText={fs.readFileSync(__dirname + '/../examples/Weather.jsx', 'utf8')} />
+          <h4>Attributes</h4>
+          <ul>
+            <li><code>weather</code> String - The type of weather that is occuring. Can be <code>cloudy</code>, <code>fog</code>, <code>hail</code>, <code>heavy-rain</code>, <code>heavy-snow</code>, <code>light-rain</code>, <code>light-snow</code>, <code>night-clear</code>, <code>night-partly-cloudy</code>, <code>partly-cloudy</code>, <code>storm</code>, <code>sunny</code> or <code>windy</code> (more to come)</li>
+            <li><code>temperature</code> Number - The current temperature</li>
+            <li><code>unit</code> String - The unit of measurement for temperature <code>c</code> (Centigrade) or <code>f</code> (Fahrenheit)</li>
```

Comment:
Would `Celsius` be better than `Centigrade`? I think that `Centigrade` is a historical name?